### ISSUE TITLE
Get individual item endpoint does not work with "bigint" identifiers

### ISSUE THREAD

**Issue by technicallyerik** [directus/api#1910]
I am attempting to get an individual item via the API against a collection that uses "bigint" for its identifiers in Directus 8.6.2:

```
GET http://test.local:8888/myproject/items/widgets/10110887584839587
```

It comes back with the response:

```
{
    "error": {
        "code": 203,
        "message": "Item with id \"1.011088758484E+16\" not found"
    }
}
```

I'm not familiar with this project to know if this a good solution, but if I get rid of this line in `RelationalTableGateway`, the API works:

```
array_walk($params, [$this, 'castFloatIfNumeric']);
```

Floats in PHP have a precision of 14 digits where an unsigned "bigint" in MySQL has 20.  It also looks like the code is casting my primary key to a CHAR anyway:

```
CAST(`' . $this->getTable() . '`.`' . $this->primaryKeyFieldName . '` as CHAR)
```

**Comment by technicallyerik** [comment]
Updating `castFloatIfNumeric` to something like this also seems to work (I'm using a 64-bit build of PHP so it can hold the identifier as an integer)

```
    public function castFloatIfNumeric(&$value, $key)
    {
        if ($key != 'table_name') {
            if(is_int($value)) {
                $value = (int)$value;
            } else if(is_float($value)) {
                $value = (float)$value;
            }
        }
    }
```

**Comment by technicallyerik** [comment]
This appears to be fixed in 8.8.1 from PR #1938
